### DISCUSSION TITLE
The ceph-mon service starts but quickly shuts down

### DISCUSSION THREAD

**Discussion by wangchao6856** [rook/rook#14547] [Q&A]
Deploying Ceph cluster version 18.2.2 on three physical machines with Ubuntu 20.04. Ceph components have been installed on each node using apt. When starting the ceph-mon service, it starts but quickly shuts down, and the  `/var/log/ceph/ceph.log` show errors：
`rocksdb: [db/db_impl/db_impl.cc:496] Shutdown: canceling all background work`
`rocksdb: [db/db_impl/db_impl.cc:704] Shutdown complete`
`ceph-mon: created monfs at /var/lib/ceph/mon/ceph-node229 for mon.node229`

Before these errors, the logs showed the startup process of RocksDB, but no issues were apparent."

Using the command `systemctl status ceph-mon@node229`, the output shows an error:
`systemd[1]: ceph-mon@node229.service: Start request repeated too quickly.`
`systemd[1]: ceph-mon@node229.service: Failed with result 'core-dump'.`
`systemd[1]: Failed to start Ceph cluster monitor daemon.`

Before starting the mon service, the admin secret was created using ceph-authtool and the /etc/ceph/ceph.conf file was configured

**Comment by travisn** [comment]
@wangchao6856 These discussions are for Rook deployments. Did you deploy with Rook? Looks like a different install.
